II.pass-tests/09-car-sales/car-sales.js

```diff
@@ -0,0 +1,26 @@
+module.exports = carsSold => {
```

Comment:
well done for the usage of `forEach` :+1: 

If you'd like to go even further, imagine if we could group all the cars by makers.
Something like
```
{
  "Ford" => [{}, {}, {}], // all ford cars
  "Hyundai" => [{}, {}]. // all Hyunday cars,
}
...
```
Then we could simply `reduce` the values down to their price. 
try looking for ways to use the function `reduce()` in StackOverflow.
[This is a good start](https://stackoverflow.com/a/40774906)